### DISCUSSION TITLE
Signed push support

### DISCUSSION THREAD

**Discussion by ajakk** [gitpython-developers/GitPython#1470] [Q&A] [answered]
Hi, is there any way to sign pushes akin to `git push --signed`? If not, could it be added?

**Comment by ajakk** [comment]
Ah, it's also possible to hack around this via git configuration.

**Comment by Byron** [comment, marked as answer]
It's possible to pass any arguments to the `push` operation as outlined [in the docs](https://gitpython.readthedocs.io/en/stable/tutorial.html#handling-remotes). Using one of the examples there, one could run `origin.push(signed=True)` I believe.
